src/ansys/geometry/core/sketch/arc.py

```diff
@@ -342,3 +347,122 @@ def from_three_points(cls, start: Point2D, inter: Point2D, end: Point2D):
 
         # Finally... you can create the arc
         return Arc(start=start, end=end, center=center, clockwise=is_clockwise)
+
+    @classmethod
+    @check_input_types
+    def from_start_end_and_radius(
+        cls,
+        start: Point2D,
+        end: Point2D,
+        radius: Union[Quantity, Distance, Real],
+        convex_arc: Optional[bool] = False,
+        clockwise: Optional[bool] = False,
+    ):
+        """
+        Create an arc from a starting point, an ending point, and a radius.
+
+        Parameters
+        ----------
+        start : Point2D
+            Starting point of the arc.
+        end : Point2D
+            Ending point of the arc.
+        radius : Union[Quantity, Distance, Real]
+            Radius of the arc.
+        convex_arc : bool, default: False
+            Whether the arc is convex or not. When ``True``, the arc is convex.
+            When ``False`` (default), the arc is concave.
```

Comment:
```suggestion
            When ``False`` (default), the arc is concave.
```
Do you need "(default)" because it is already defined after the data type. I'd put the behavior for the default (``False``) first, followed by the behavior for ``True``.

Reply:
Agreed. Let me change that. Thanks @PipKat!